libs/ui-watt/src/lib/components/stepper/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { WattStepperButtonNextDirective } from './watt-stepper-button-next';
```

Comment:
I'm missing some docs or showcases of these new `watt` components :)

Reply:
They cannot live alone just yet :-)

Reply:
I added them, so we could use our own watt-buttons :-)